docs/contributing.md

```diff
@@ -1,11 +1,22 @@
 # How to Contribute
 
-ConfigSync follows the [Kubernetes contribution process].  If you
-are looking to contribute to this project please fork it, create a change
-and then submit a PR against the `main` branch which is the HEAD of development.
+Contributions are greatly appreciated. The maintainers of Config Sync monitor issues
+and PRs and strive to provide timely feedback and reviews. For any questions or 
+live discussions please ask questions in the #kpt channel on [slack].
```

Comment:
I think @janetkuo mentioned that she would prefer async comms on discussion groups or issues.  Is that something we want to state?